packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -37,6 +37,10 @@ const propTypes = {
    */
   // eslint-disable-next-line react/forbid-prop-types
   dropdownAttrs: PropTypes.object,
+  /**
+   * The id of the input field.
+   */
+  id: PropTypes.string,
```

Comment:
Should this prop name be scoped with `inputId`? Currently if I was using select, I could pass an `id` to the component via `customProps`. I'd be a little surprised if this `id` value was suddenly being spread on the input.

Reply:
Changed in 22b163c